prov/verbs/src/fi_verbs.h

```diff
@@ -289,6 +289,14 @@ struct fi_ibv_eq {
 		 * consider using an internal PEP listener for handling the
 		 * internally processed reciprocal connections. */
 		uint16_t		pep_port;
+
+		/* SIDR request/responses are a two-way handshake; therefore,
+		 * we maintain an RB tree of SIDR accept responses, so that if
+		 * a response is lost, the subsequent retried request can be
+		 * detected and the original accept response resent. Note, that
+		 * reject responses can be passed to RXM and will be rejected
```

Comment:
nit: I think this should be 'rejected requests' instead of responses.

Reply:
Agreed, I will update the comment in the associated commit. Thanks.